### PR TITLE
feat(youtube): `default-video-quality` patch

### PR DESCRIPTION
**Pull request by johnconner122**
depends on:

 - https://github.com/revanced/revanced-integrations/pull/225

### CODE REVIEW

File: src/main/kotlin/app/revanced/patches/youtube/misc/video/quality/patch/RememberVideoQualityPatch.kt [hunk 1]
```diff
@@ -1,88 +0,0 @@
-package app.revanced.patches.youtube.misc.video.quality.patch
```
Comment (oSumAtrIX) [inline src/main/kotlin/app/revanced/patches/youtube/misc/video/quality/patch/RememberVideoQualityPatch.kt:1]: This PR gets rid of the functionality to remember a quality. Instead it should only offer to change it manually in addition to offering the option to locking it. That way, you can either remember the last quality, change it manually in the settings and lock it optionally.

**Review by oSumAtrIX — CHANGES_REQUESTED** [review]

File: src/main/kotlin/app/revanced/patches/youtube/misc/video/quality/patch/RememberVideoQualityPatch.kt (hunk 1 above)
Comment (johnconner122) [reply src/main/kotlin/app/revanced/patches/youtube/misc/video/quality/patch/RememberVideoQualityPatch.kt:1]: It is handled in DefaultVideoQualityPatch

**Comment by kazimmt** [comment]
Following Silently

**Comment by SodaWithoutSparkles** [comment]
For reference: #826's successor

File: src/main/kotlin/app/revanced/patches/youtube/misc/video/quality/patch/DefaultVideoQualityPatch.kt [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+                        )
+                    ),
+                    ListPreference(
+                        "revanced_default_video_quality_wifi",
+                        StringResource(
+                            "revanced_default_video_quality_wifi_title",
+                            "Change video quality on Wi-Fi network"
+                        ),
+                        ArrayResource("revanced_video_quality_wifi_entries", keys),
+                        ArrayResource("revanced_video_quality_wifi_entryValues", values),
```
Comment (oSumAtrIX) [inline src/main/kotlin/app/revanced/patches/youtube/misc/video/quality/patch/DefaultVideoQualityPatch.kt:86]: This key should be named in snake casing:

```suggestion
                        ArrayResource("revanced_video_quality_wifi_entry_values", values),
```

Make sure to apply the change throughout the integrations as well.

**Review by oSumAtrIX — CHANGES_REQUESTED** [review]

**Comment by oSumAtrIX** [comment]
Your repository does not fork from ours, which is why this PR can not work. Please delete your remote repository. Afterwards fork this repository again and force push your local `default-video-quality` branch over it. Reopen the PR.

**Pull request closed without merging**